docker/mysql-to-postgres/bin/migrate-mysql-to-postgres

```diff
@@ -177,22 +186,16 @@ if needs_fulltext_migration
     exit 1
   end
 
-  _, extract_status = run "bundle exec rake attachments:extract_fulltext_where_missing", record_output: false
+  _, extract_status = run "bundle exec rake attachments:schedule_text_fulltext_extraction_where_missing", record_output: false
```

Comment:
Is `schedule_text_fulltext_extraction_where_missing` the correct name here?

Reply:
I think so, because `schedule_text_fulltext_extraction_where_missing` will trigger with `run_now: false` and in turn generate jobs? Tested this on the tmm dump and it created 2000 jobs correctly

Reply:
The logic is correct, I'm just wondering if that rake task is correctly named? I find `schedule_text_fulltext_extraction_where_missing` a bit convoluted. Why not just `schedule_fulltext_extraction_where_missing` for instance?

Reply:
Of course I agree, thanks - I missed that

Reply:
Fixed